pkg/rc/farm.go

```diff
@@ -53,7 +53,7 @@ type Farm struct {
 
 	children map[fields.ID]childRC
 	childMu  sync.Mutex
-	session  kp.Session
+	session  consul.Session
```

Comment:
I like this change because this sort of thing is now a very visible sign of tight coupling.